addons/mail/static/tests/discuss/core/composer.test.js

```diff
@@ -116,8 +116,8 @@ test("html composer: send a message in a channel", async () => {
         editable: document.querySelector(".o-mail-Composer-html.odoo-editor-editable"),
     };
     await htmlInsertText(editor, "Hello");
-    await contains(".o-mail-Composer-html.odoo-editor-editable", { text: "Hello" });
+    await contains(".o-mail-Composer-html.odoo-editor-editable:text('Hello')");
     await click(".o-mail-Composer button[title='Send']:enabled");
     await click(".o-mail-Message[data-persistent]:contains(Hello)");
-    await contains(".o-mail-Composer-html.odoo-editor-editable", { text: "" });
+    await contains(".o-mail-Composer-html.odoo-editor-editable:text('Hello')", { count: 0 });
```

Comment:
This change doesn't look correct: the intention of `:text()` is to have exact check, including checking theres's no text content. So should have been `:text('')` without implicit `count: 1`.

Reply:
:text('') is not working so that's the issue. I use `textContent` in some places